src/main/java/com/kobylynskyi/graphql/codegen/utils/SerialVersionUIDGenerator.java

```diff
@@ -0,0 +1,16 @@
+package com.kobylynskyi.graphql.codegen.utils;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class SerialVersionUIDGenerator {
+
+    /**
+     * Generate random long
+     *
+     * @return random long
+     */
+    public long randomLong() {
+        return ThreadLocalRandom.current().nextLong();
```

Comment:
I have concerns about this.

AFAIU, a different serial version UID will be generated each time the code generation is triggered.
This means that even if the schema doesn't change, all classes will get a new UID and thus be declared "incompatible" (that's the meaning of the serial UID).

I think we should rely on the compiler-generated one without attempting to generate our own. It may be safer

Reply:
Thanks for your feedback. I also had concerns whether to define an explicit serialVersionUID or not.
But the following phrase overweight me:
“ However, it is strongly recommended that all serializable classes explicitly declare serialVersionUID values, since the default serialVersionUID computation is highly sensitive to class details that may vary depending on compiler implementations, and can thus result in unexpected InvalidClassExceptions during deserialization.”

Reply:
@kobylynskyi what the sentence means essentially is that if you make a backwards *compatible* change in the class, due to the sensitivity of the generation algorithm, you may get a different version UID anyway. This will trigger `InvalidClassException`s at deserialization time even if the deserialization would in fact have worked.

But in our case, generating it randomly is even worse than that: we will *always* generate incompatible versions even if no changes are made in the class at all. So classes generated by 2 different builds will always be incompatible and trigger `InvalidClassException`s.
So I guess for the use case of generated code, not generating a serial version UID is the lesser evil.

We could try to attempt generating a serial version UID based on the actual schema and details of the class generated (so that the same class yields the same version ID), but that would basically be reimplementing the auto-generation of the serialization runtime, so I guess we can assume they already did a good job.

The only superior approach would be manual attribution of ID (which is the recommended way for classes you write manually and own), but it doesn't suit our use case.

Reply:
Regarding custom directive - that's a nice idea! I think we can consider this in the future.